packages/connect-form/src/components/advanced-options-tabs/authentication-tab/authentication-x509.tsx

```diff
@@ -1,9 +1,12 @@
+import { Banner, BannerVariant } from '@mongodb-js/compass-components';
 import React from 'react';
 
 function AuthenticationX509(): React.ReactElement {
   return (
     <>
-      <p>X.509</p>
+      <Banner variant={BannerVariant.Info}>
+        X.509 Authentication type requires a Client Certificate to work.
```

Comment:
Maybe we can guide the user a bit more here, I think we've chatted in the past on having links to the TLS/SSL tab, which is something we might want to do. For the time being maybe we can mention the tab until we add that feature?
`X.509 Authentication type requires a Client Certificate to work. This can be set in the TLS/SSL tab.`

Reply:
Good idea, updated!